Makefile

```
@@ -6,19 +6,41 @@ ifneq ($(OS),Windows_NT)
 	OS := $(shell uname)
 endif
 
+.DEFAULT_GOAL := help
+
+.ONESHELL:
+
+.SHELLFLAGS = -ec
+
 .PHONY: help
 help: ## Show this help.
-	@printf "%-30s %s\n" "Target" "Description"
-	@printf "%-30s %s\n" "------" "-----------"
-	@fgrep " ## " $(MAKEFILE_LIST) | fgrep -v grep | awk -F ': .*## ' '{$$1 = sprintf("%-30s", $$1)} 1'
+	@printf "%-41s %s\n" "Target" "Description"
+	@printf "%-41s %s\n" "------" "-----------"
+	@fgrep " ##" $(MAKEFILE_LIST) | fgrep -v grep | sed 's/:[^#]*/ /' | awk -F '[: ]*' \
+	'{
+		if ($$2 == "##")
+		{
+			$$1=sprintf("%-40s", $$1);
+			$$2="";
+			print $$0;
+		}
+		else
+		{
+			$$1=sprintf(" └%-38s", $$1);
+			$$2="";
+			print $$0;
+		}
+	}'
+
+---------------- : ## ----------------
 
 .PHONY: setup
 setup: ## Set up a development environment
 ifeq ($(OS),Linux)
-	sudo snap install codespell ruff shellcheck
-	sudo snap install --classic --beta astral-uv
-	sudo snap alias astral-uv.uv uv
-	sudo snap alias astral-uv.uvx uvx
+	sudo snap install codespell
+	sudo snap install ruff
+	sudo snap install shellcheck
+	sudo snap install --classic astral-uv
```

I can't add a suggestion that includes the next line down in https://github.com/canonical/starbase/pull/265/files#r1797066230, so new comment thread I guess.

What about:

```suggestionchanges=""
	changes="${changes} `sudo snap install --no-wait codespell`"
	changes="${changes} `sudo snap install --no-wait ruff`"
	changes="${changes} `sudo snap install --no-wait shellcheck`"
	changes="${changes} `sudo snap install --no-wait astral-uv`"
	for change in ${changes}; do
	  snap watch ${change}
	done
```

This starts all the installs in parallel and gives a spot to do other installs while the snaps are installing.

---

Love it, I'll do that!